src/main/java/com/derongan/minecraft/deeperworld/synchronization/SectionSyncListener.kt

```diff
@@ -103,6 +102,14 @@ object SectionSyncListener : Listener {
         blockEvent.newState.location.sync()
     }*/
 
+    @EventHandler
+    fun BlockPhysicsEvent.sync() {
```

Comment:
Still scared about listening to physics event. What is this fixing by being cancelled?

Reply:
Inside the section overlaps Signs, Rails, Torches etc will not pop off automatically (fixing dupes), you have to mine them manually.